src/core_pdf/impl/engine/parse/capture.py

```
@@ -226,17 +206,21 @@ def internal_glyph_evidence_fields(
     unsupported = 0
     low_confidence = 0
-    visible = 0
     semantic_characters = 0
-    sources: Counter[str] = Counter()
     glyph_count = 0
     previous_decoder: object | None = None
     learned: dict[bytes, str] | None = None
-    for glyph_text, glyph_visible, decoder, code_bytes, unicode_source, confidence in glyph_fields:
+    for (
+        glyph_text,
+        ignored_visible,
```

Ruff (0.16.2)

[warning] Loop control variable `ignored_visible` not used within loop body

Rename unused `ignored_visible` to `_ignored_visible`

(B007)